### ISSUE TITLE
How to disable typescript plugin or typescript server for vue(lang="ts") file?

### ISSUE THREAD

**Issue by django-wong** [Microsoft/TypeScript-Sublime-Plugin#665]
I can disable typescript plugin globally but I still want this plugin enabled for my other pure typescript project or file

<img width="725" alt="screen shot 2018-05-18 at 11 22 15 am" src="https://user-images.githubusercontent.com/4019240/40214623-5d15b5b4-5a8e-11e8-9298-e737f8e50838.png">

**Comment by muweigg** [comment]
+1 >_<

**Comment by django-wong** [comment]
Try this

Alter file `typescript/libs/view_helpers.py` to update function `is_typescript` like this:
```python
def is_typescript(view):
    """Test if the outer syntactic scope is 'source.ts' or 'source.tsx' """
    filename = view.file_name()
    blocklist = [".vue", ".html"]
    name, extension = os.path.splitext(filename)
    if not filename:
        return False

    if extension in blocklist:
        return False

    try:
        location = view.sel()[0].begin()
    except:
        return False

    return (view.match_selector(location, 'source.ts') or
            view.match_selector(location, 'source.tsx'))
```

**Comment by muweigg** [comment]
@django-wong Good job, It works fine, thank you.

**Comment by nehbit** [comment]
Thanks @django-wong. This saved my bacon as well, I was on the verge of giving up and stop using Typescript for new Vue files from now on.

**Comment by django-wong** [comment]
@nehbit that's true, also use [LSP](https://github.com/tomv564/LSP) + [vue-language-server](https://github.com/vuejs/vetur/tree/master/server) can bring the code intellisence back

**Comment by DanielRosenwasser** [comment]
Sorry for the delay, the fix is out at #705.

**Comment by nehbit** [comment]
Thank you!
